Lib/defcon/objects/font.py

```diff
@@ -884,6 +893,8 @@ def _get_identifiers(self):
     # object as dirty.
 
     def _get_guidelines(self):
+        if self._info is None:
+            self.info
```

Comment:
We are not throwing away anything? the info object is stored in self._info.

self.info.guidelines just calls font.guidelines so it would be kind of a roundabout.